### PR TITLE
Update gajim, python3-nbxmpp, gajim-omemo

### PR DESCRIPTION
**Pull request by teldra**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by dexgs** [comment]
Can the version of gajim-omemo also be bumped?

File: srcpkgs/gajim/template [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
 build_style=python3-module
 hostmakedepends="gettext python3-setuptools"
-depends="python3-gobject python3-nbxmpp python3-setuptools python3-Pillow
- python3-cairo python3-precis-i18n python3-keyring python3-packaging
- python3-css-parser farstream python3-dbus python3-openssl"
+depends="python3-gobject python3-setuptools python3-Pillow
+	python3-cairo python3-dbus libsecret gupnp-igd gsound geoclue2 libnm
+	python3-nbxmpp python3-openssl python3-css-parser python3-keyring
+	python3-precis-i18n python3-packaging python3-setuptools libsoup farstream
+	gst-plugins-base1 gst-plugins-ugly1 gst-libav gtksourceview4"
```
Comment (ericonr) [inline srcpkgs/gajim/template:11]: Please keep indentation as 1 space here

**Comment by ericonr** [comment]
gajim-omemo is also in https://github.com/void-linux/void-packages/pull/37117/commits/79383ba6995163fd778dec1b0124d29c24c501a0

@chili-b I will count your PR as additional testing for this too, thank you

**Comment by dexgs** [comment]
https://gajim.org/post/2022-05-21-gajim-1.4.1-released/ gajim 1.4.1 is now out

File: srcpkgs/gajim/template (hunk 1 above)
Comment (teldra) [reply srcpkgs/gajim/template]: Well, sure.. new editor ^^

**Comment by classabbyamp** [comment]
gajim 1.4.5 is now available

File: srcpkgs/gajim/template [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
 hostmakedepends="gettext python3-setuptools"
-depends="python3-gobject python3-nbxmpp python3-setuptools python3-Pillow
- python3-cairo python3-precis-i18n python3-keyring python3-packaging
- python3-css-parser farstream python3-dbus python3-openssl"
+depends="python3-gobject python3-setuptools python3-Pillow
+ python3-cairo python3-dbus libsecret gupnp-igd gsound geoclue2 libnm
+ python3-nbxmpp python3-openssl python3-css-parser python3-keyring
+ python3-precis-i18n python3-packaging python3-setuptools python3-distro
+ python3-netifaces python3-pyasn1 libsoup farstream gst-plugins-base1
+ gst-plugins-ugly1 gst-libav gtksourceview4"
```
Comment (CameronNemo) [inline srcpkgs/gajim/template:12]: where did you get this list of dependencies?

Comment (teldra) [reply srcpkgs/gajim/template:12]: It is a mixture from setup.cfg and archlinux pkg and https://dev.gajim.org/gajim/gajim/. Did i miss something?

**Pull request merged**